web/src/main/resources/templates/show.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en" xmlns:th="http://www.w3.org/1999/xhtml">
+<head>
+    <meta charset="UTF-8">
+    <title>Title</title>
+</head>
+<body>
+<div class="message" th:if="message">
+    <h2 th:text="${message}"/>
+</div>
+<div th:if="${customers}">
```

Comment:
Very nice!
So now link a new css file containing HTML styling and try to use it in your HTML elements :)
A few simple uses would be good for a start

Reply:
Can i use bootstrap?